reference/sifive-hifive-unmatched.rst

```diff
@@ -2,18 +2,178 @@
+
+  - :download:`ubuntu-24.04-preinstalled-server-riscv64+unmatched.img.xz <https://cdimage.ubuntu.com/releases/24.04/release/ubuntu-24.04-preinstalled-server-riscv64+unmatched.img.xz>`
+
+* Ubuntu 24.04 (Noble Numbat) live server (see instructions below):
+
+  - :download:`ubuntu-24.04-live-server-riscv64.img.gz <https://cdimage.ubuntu.com/releases/24.04/release/ubuntu-24.04-live-server-riscv64.img.gz>`
+
+* Ubuntu 23.10 (Mantic Minotaur) pre-installed server:
+
+  - :download:`ubuntu-23.10-preinstalled-server-riscv64+unmatched.img.xz <http://cdimage.ubuntu.com/releases/23.10/release/ubuntu-23.10-preinstalled-server-riscv64+unmatched.img.xz>`
+
+* Ubuntu 23.10 (Mantic Minotaur) live server (see instructions below):
+
+  - :download:`ubuntu-23.10-live-server-riscv64.img.gz <http://cdimage.ubuntu.com/releases/23.10/release/ubuntu-23.10-live-server-riscv64.img.gz>`
+
+* Ubuntu 22.04 (Jammy Jellyfish) pre-installed server:
+
+  - :download:`ubuntu-22.04.4-preinstalled-server-riscv64+unmatched.img.xz <http://cdimage.ubuntu.com/releases/22.04/release/ubuntu-22.04.4-preinstalled-server-riscv64+unmatched.img.xz>`
+
+* Ubuntu 22.04 (Jammy Jellyfish) live server (see instructions below):
+
+  - :download:`ubuntu-22.04.4-live-server-riscv64.img.gz <http://cdimage.ubuntu.com/releases/22.04/release/ubuntu-22.04.4-live-server-riscv64.img.gz>`
+
 
 Using the pre-installed server image
 ====================================
 
+#. Flash the pre-installed server image to a microSD card (see
+   :doc:`/how-to/flash-images`)
+
+#. Insert the microSD card into the board
+
+#. Set the boot source to the microSD card (see `Boot source selection`_)
+
+#. Connect the USB UART cable to the :term:`UART` header (see `UART console`_
+   and :doc:`/how-to/uart-console`)
+
+#. Power on the board
+
+#. Wait for an output line confirming that `cloud-init`_ has finished running;
+   this service is responsible for generating SSH keys, and creating the
+   default user:
+
+   .. code-block:: text
+
+       [   35.682018] cloud-init[909]: Cloud-init v. 24.1.3-0ubuntu3 finished at Tue, 23 Apr 2024 07:44:59 +0000. Datasource DataSourceNoCloud [seed=/var/lib/cloud/seed/nocloud-net][dsmode=net].  Up 35.65 seconds
+
+#. Login with the user *ubuntu* and the default password *ubuntu*; you will be
+   asked to choose a new password
+
 
 Using the live server image
 ===========================
 
+The live installer image is used to install Ubuntu to an :term:`NVMe` drive
+attached to the M.2 slot on the board.
+
+#. Flash the pre-installed server image to a microSD card (see
+   :doc:`/how-to/flash-images`)
+
+#. Insert the microSD card into the board
+
+#. Set the boot source to the microSD card (see `Boot source selection`_)
+
+#. Connect the USB UART cable to the :term:`UART` header (see `UART console`_
+   and :doc:`/how-to/uart-console`)
+
+#. Power on the board
+
+#. When "Hit any key to stop autoboot" is displayed, press :kbd:`Enter`
```

Comment:
These U-Boot CLI commands are only needed if an operating system is already installed on the NVMe drive.

Reply:
Added an update to clarify this